leaflet/static/leaflet/leaflet.extras.js

```diff
@@ -116,6 +116,11 @@ L.Map.DjangoMap = L.Map.extend({
                 minZoom: this.options.minZoom,
                 maxZoom: this.options.maxZoom
             });
+            // Google maps Plugin
+            if (layer.url.substr(0,6) == 'GOOGLE') {
```

Comment:
This seems a bit fragile.
I would rather test with a regular expression : `` if (/^GOOGLE/.test(layer.url)) {``